flight/Libraries/inc/morsel.h

```diff
@@ -0,0 +1,33 @@
+// Morse Library header
```

Comment:
Okay. Up to you if you want to do it now. Probably can wait for a future occasion when it's necesssary.